libs/blocks/quiz-entry/quiz-entry.js

```diff
@@ -199,18 +211,43 @@ const App = ({
     setSelectedCards(selected);
   };
 
-  const onMLInput = (event) => {
+  const onMLInput = async (event) => {
     const inputValue = event.target.value;
     setMLInputUsed(true);
+    const { mlDetails } = mlData;
+    const data = await getSuggestions(mlDetails['ac-endpoint'], mlDetails['ac-client-id'], inputValue, mlDetails['ac-scope']);
+    if (data && data.suggested_completions && data.suggested_completions.length > 0) {
+      setSuggestions(data.suggested_completions.slice(0, 5));
+      setShowPopover(true);
+    }
+
+    document.querySelector('#ml-field-clear').classList.toggle('hidden', !inputValue.length);
+
     if (inputValue.length === 0) {
       setMLInputUsed(false);
+      setSuggestions([]);
+      setShowPopover(false);
     }
   };
 
   const onMLEnter = (event) => {
     if (event.keyCode === enterKeyCode) continueQuiz();
   };
 
+  const onSuggestionClick = (suggestion) => () => {
```

Comment:
Can we add a way to put focus back into the input field after a suggestion is selected? without restoring focus, onMLEnter doesn't work - you can't just select from autocomplete and press enter to continue